src/features/onboarding/pages/OnboardingPage.tsx

```diff
@@ -14,6 +16,17 @@ export interface OnboardingPageProps {
   onGoBack?: () => void
 }
 
+const buttonsContainerAnimation = {
```

Comment:
= Object.assign(opacityAnimation, buttonScaleUpAnimation)

opacityAnimation = {from:{opacity:0},to:{opacity:1}}
...

Reply:
[Object.assign](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) ne garde que la dernière valeur si il y a des clés communes

Reply:
Ok mais il existe un système de merge d'objet qui pourrait être utilisé non ?

Reply:
Genre
const mergeAnimation = (a,b) => {
	const from = Object.assign({}, a.from, b.from);
	const to = Object.assign({}, a.to, b.to);
	return Object.assign({}, {from:from}, {to:to});
}